localstack/utils/bootstrap.py

```diff
@@ -44,10 +44,9 @@
 # maps from API names to list of other API names that they depend on
 API_DEPENDENCIES = {
     "dynamodb": ["dynamodbstreams"],
-    "dynamodbstreams": ["kinesis"],
+    "dynamodbstreams": ["kinesis", "dynamodb"],
```

Comment:
I suppose the dependency is the other way around, if somebody wants to use `dynamodbstreams` then they have to enable `dynamodb`, I don't think we should enable it for them.

Reply:
Also, I don't think this is blocking or anything, we can update the list later on!

Reply:
that makes sense. i will remove it again :)